package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "leaderboards",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Kindly make sure `npm run ?` scripts like dev, start are in `package.json`, this will enable you to start the server 👍🏿 

```json
 "scripts": {
    "dev": "webpack --mode development",
    "start": "webpack --config webpack.config.js --watch"
  }
```